drivers/clock_control/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ zephyr_library_sources_ifdef(CONFIG_CLOCK_CONTROL_LPC11U6X            clock_cont
 zephyr_library_sources_ifdef(CONFIG_CLOCK_CONTROL_MCHP_XEC            clock_control_mchp_xec.c)
```

Comment:
@LaurentiuM1234 would also like to see in the commit message what CCM v3 driver brings in over v2.

Reply:
Sure, can do so. The main advantages of CMM v3 over the pervious versions are:

1. Scalability + readability. Since pervious versions heavily rely on macros, adding support for it on more SoCs will heavily impact its readability as you'd need to use even more macros. This is because the HAL API may change from SoC to SoC.
2. Flexibility. SoC developers may choose to implement the clock control operations for their SoC however they see fit. 
3. Ability to set a clock's frequency.